moonbeam-types-bundle/README.md

```diff
@@ -79,12 +80,12 @@ Running the latest TypeScript version will not work.
 Ultimately it is necessary to use the new type naming as the previous one won't be supported, but
 you can import `typesBundleDeprecated` to buy yourself some time.
 
-* moonbeam-types-bundle v1.x.x will break on runtime upgrade 900
-(planned Thursday 18th November 2021 on Moonriver)
-* moonbeam-types-bundle v2.x.x `typesBundleDeprecated` (using previous naming case) 
-will **break on runtime 1000**
-* **moonbeam-types-bundle v2.x.x** `typesBundlePre900` (using new naming case) 
-will be **maintained**.
```

Comment:
Please prepare a follow up PR with these linter changes, where they can be discussed.

Reply:
I've just reverted the markdown changes that modified this